x-pack/plugins/ml/public/application/model_management/force_stop_dialog.tsx

```diff
@@ -41,46 +71,69 @@ export const ForceStopModelConfirmDialog: FC<ForceStopModelConfirmDialogProps> =
         { defaultMessage: 'Stop' }
       )}
       buttonColor="danger"
+      confirmButtonDisabled={model.deployment_ids.length > 1 && selectedDeploymentIds.length === 0}
     >
-      <FormattedMessage
-        id="xpack.ml.trainedModels.modelsList.forceStopDialog.pipelinesWarning"
-        defaultMessage="You can't use these ingest pipelines until you restart the model:"
-      />
-      <ul>
-        {Object.keys(model.pipelines!)
-          .sort()
-          .map((pipelineName) => {
-            return <li key={pipelineName}>{pipelineName}</li>;
-          })}
-      </ul>
+      {model.deployment_ids.length > 1 ? (
+        <EuiCheckboxGroup
+          legend={{
+            display: 'visible',
+            children: (
+              <FormattedMessage
+                id="xpack.ml.trainedModels.modelsList.forceStopDialog.selectDeploymentsLegend"
+                defaultMessage="Select deployments to stop"
+              />
+            ),
+          }}
+          options={options}
+          idToSelectedMap={checkboxIdToSelectedMap}
+          onChange={onChange}
+        />
+      ) : null}
+
+      {isPopulatedObject(model.pipelines) ? (
```

Comment:
Nit - how does this look with a spacer to give more separation between this warning message and the checkboxes above?

<img width="587" alt="image" src="https://user-images.githubusercontent.com/7405507/234260396-e5f14bde-6959-44ed-a573-a2a103a68c8b.png">

Reply:
Updated in https://github.com/elastic/kibana/pull/155375/commits/47c791cf47768024aadc3f17e34cbe9e1f164da7

Reply:
The callout you added here looks good.

<img width="565" alt="image" src="https://user-images.githubusercontent.com/7405507/234307912-8eb1aefd-d7c7-4a14-b425-71a3c1ad5d51.png">